src/upgrader.js

```diff
@@ -309,8 +310,8 @@ class Upgrader {
       },
       newStream: newStream || errConnectionNotMultiplexed,
       getStreams: () => muxer ? muxer.streams : errConnectionNotMultiplexed(),
-      close: async (/** @type {Error | undefined} */ err) => {
-        await maConn.close(err)
+      close: async () => {
+        await maConn.close()
```

Comment:
Looking at the [use of this callback](https://github.com/libp2p/js-libp2p-interfaces/blob/master/src/connection/connection.js#L224) no error is passed in, which is reflected in the [callback typedef](https://github.com/libp2p/js-libp2p-interfaces/blob/master/src/connection/connection.js#L33) so this function probably shouldn't accept an error param?